sympy/polys/matrices/dense.py

```diff
@@ -195,6 +495,33 @@ def ddm_idet(a, K):
 
 
 def ddm_iinv(ainv, a, K):
+    """ainv  <--  inv(a)
+
+    Compute the inverse of a matrix $a$ over a field $K$ using Gauss-Jordan
+    elimination. The result is stored in $ainv$.
+
+    Uses division in the ground domain which should be an exact field.
+
+    Examples
+    ========
+
+    >>> from sympy.polys.matrices.ddm import ddm_iinv, ddm_imatmul
+    >>> from sympy import QQ
+    >>> a = [[QQ(1), QQ(2)], [QQ(3), QQ(4)]]
+    >>> ainv = [[None, None], [None, None]]
+    >>> ddm_iinv(ainv, a, QQ)
+    >>> ainv
+    [[-2, 1], [3/2, -1/2]]
+    >>> result = [[QQ(0), QQ(0)], [QQ(0), QQ(0)]]
```

Comment:
```suggestion
    >>> result = []
```

Reply:
There is no need for user to worry about what to put in for elements of `ainv` - providing an mutable is sufficient since it will be filled appropriately (and there is no check, anyway, for the shape of `ainv`).